cmd/observe/observe_filter.go

```diff
@@ -339,10 +341,29 @@ func (of *observeFilter) set(f *filterTracker, name, val string, track bool) err
 			})
 		}
 		f.apply(func(f *pb.FlowFilter) {
-			// TODO: auto-check L7?
 			f.HttpStatusCode = append(f.HttpStatusCode, val)
 		})
 
+	case "http-method":
+		if wipe {
```

Comment:
Someone can correct me, but I don't think `wipe` is relevant to `default nil` use-cases of filters right?. 

Perhaps we can look into rewriting this a bit so it's clearer when to use `wipe`.

Reply:
Correct! I assume @twpayne copied this over from `http-status`, which also pontelessly wipes the value. This is not needed in any of the `http-*` flags.

https://github.com/cilium/hubble/blob/903e3c8b9b03aae88c331900c5cb45e4cd1fdfc7/cmd/observe/observe_filter.go#L40-L42

@twpayne Can you remove all the `wipe` from all `http-*` flags? It's technically an unrelated cleanup to this PR, but will make it a bit cleaner.

Reply:
Done.